t/jobspec/valid/example1.yaml

```diff
@@ -0,0 +1,19 @@
+version: 1
+resources:
+  - type: node
+    count: 4
+    with:
+      - type: slot
+        count: 1
+        label: default
+        with:
+          - type: core
+            count: 2
+tasks:
+  - command: app
+    slot: default
+    count:
+      per_slot: 1
+attributes:
+  system:
+    duration: 1 hour
```

Comment:
Github diff noted that this file is missing a final newline. Not a big deal if it works (or if it is on purpose), but I thought I'd point it out because it is different from all the other yaml files here.

Reply:
I want to leave that one as-is.